src/catalog/database_catalog.cpp

```diff
@@ -1269,12 +1269,15 @@ bool DatabaseCatalog::CreateIndexEntry(transaction::TransactionContext *const tx
       pg_index_all_cols_prm_[postgres::INDISEXCLUSION_COL_OID]))) = schema.is_exclusion_;
   *(reinterpret_cast<bool *>(indexes_insert_pr->AccessForceNotNull(
       pg_index_all_cols_prm_[postgres::INDIMMEDIATE_COL_OID]))) = schema.is_immediate_;
+  // TODO(Matt): these should actually be set later based on runtime information about the index. @yeshengm
```

Comment:
Sure. It's related to the pg_index & pg_indexes catalog.

Reply:
The comment is referring to the fields below: live, ready, valid.